src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,65 @@
+package lotto.model;
+
+import java.util.List;
+import java.util.Set;
+
+public class Lotto {
+    public static final int LOTTO_SIZE = 6;
+    private static final String WRONG_LOTTO_SIZE = "로또는 6개의 숫자로 이루어져야 합니다.";
+    private static final String NO_DUPLICATE_NUMBER = "로또 숫자는 중복되지 않아야 합니다.";
+    private final List<LottoNo> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        this.numbers = sort(numbers).stream()
+                .map(LottoNo::new)
+                .toList();
```

Comment:
```suggestion
        this.numbers = numbers.stream()
                .map(LottoNo::new)
                .sorted()
                .toList();
```
스트림이 두 번 반복되다 보니 비효율적일 것  같습니당

Reply:
오... 놓친 부분이네요
알려주셔서 감사합니다!!